faq.html

```diff
@@ -123,7 +123,7 @@ <h2 id="kernelwriting">Writing Parcels Kernels</h2>
   <p></p>
   However, there are some key limitations to the Kernels that everyone who wants to write their own should be aware of:
   <ul>
-    <li>Every Kernel must be a function with the following (and only those) arguments: <code class="language-python">(particle, fieldset, time, dt)</code></li>
+    <li>Every Kernel must be a function with the following (and only those) arguments: <code class="language-python">(particle, fieldset, time)</code></li>
```

Comment:
Shouldn't we advertise here that this is new since v2.0 (especially in the faq) ?